pkg/query/stats.go

```diff
@@ -2,10 +2,10 @@ package query
 
 import (
 	"fmt"
+	"github.com/HdrHistogram/hdrhistogram-go"
```

Comment:
@RobAtticus it was placed ( via go fmt ) exacly there. Due to aphabetic order if I'm not mistaken )

Reply:
Right, but if you add a line break it will then alphabetize the blocks independently. This way we keep std lib dependencies separate from 3rd party deps

Reply:
oh, I was not aware of it :) Addressed in the latest commit.